api/server/router/image/backend.go

```diff
@@ -26,7 +26,7 @@ type imageBackend interface {
 	ImageHistory(ctx context.Context, imageName string) ([]*image.HistoryResponseItem, error)
 	Images(ctx context.Context, opts types.ImageListOptions) ([]*types.ImageSummary, error)
 	GetImage(ctx context.Context, refOrID string, options image.GetImageOpts) (*dockerimage.Image, error)
-	TagImage(ctx context.Context, imageName, repository, tag string) (string, error)
+	TagImageWithReference(ctx context.Context, id dockerimage.ID, newRef reference.Named) error
```

Comment:
🚲 🏠 Should we just rename `TagImageWithReference()` to `TagImage()`. I feel like `TagImageWithReference` was named like that because there already was a `TagImage()` but now that that's gone, we may as well use the more "to the point" name.

Reply:
:+1: on this